docs/sources/flow/reference/components/prometheus.exporter.windows.md

```diff
@@ -180,7 +180,7 @@ For a server name to be included, it must match the regular expression specified
 ### text_file block
 Name | Type     | Description | Default | Required
 ---- |----------| ----------- | ------- | --------
-`text_file_directory` | `string` | The directory containing the files to be ingested. | `C:\Program Files\windows_exporter\textfile_inputs` | no
+`text_file_directory` | `string` | The directory containing the files to be ingested. | `C:\Program Files\Grafana Agent Flow\textfile_inputs` | no
```

Comment:
Digging through the code, it looks like the default path is where the Agent is launched from joined with `textfile_inputs`.

https://github.com/grafana/windows_exporter/blob/3391dcec6884e64b0f6face500805c9fbc406b31/collector/textfile.go#L364-L367

Are you running Grafana Agent Flow as system service here? I'm not familiar with running the agent on Windows, so I'm ccing more involved people here, do we think we should default to a path within the agent's `--storage.path`?

cc @mattdurham @ptodev

Reply:
the documentation change change just reflect the current code, when running the `prometheus.exporter.windows` without the `text_file` block.
it might be an option to point the default path to another location

```
prometheus.exporter.windows "default" {
  enabled_collectors = ["textfile"]
}
```